team-props/static-analysis/detekt-config.yml

```diff
@@ -141,5 +144,17 @@ comments:
     active: true
   UndocumentedPublicClass:
     active: false
+    searchInNestedClass: true
+    searchInInnerClass: true
+    searchInInnerInterface: true
   UndocumentedPublicFunction:
     active: false
+
+# *experimental feature*
```

Comment:
No, I'm just merging in from the mainline ruleset because they keep changing quite fast. When the ruleset stabilises I'll make a cleanup pass.